### PR TITLE
osx remote access config

### PR DESCRIPTION
**Pull request by bitvirago**

### CODE REVIEW

File: setupfiles/bootstrap_remote_access.sh.tmpl [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+#restart ssh service
+sudo launchctl stop com.openssh.sshd
+sudo launchctl start com.openssh.sshd
+
+
+### Setup ngrok
+# create ngrok config
+NGROK_CONF=$(cat <<EOC
+        authtoken: {{.Token}}
```
Comment (ghaabor) [inline setupfiles/bootstrap_remote_access.sh.tmpl]: Misaligned tabs

File: setupfiles/bootstrap_remote_access.sh.tmpl [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        log_format: logfmt
+        log: /var/log/ngrok.log
+tunnels:
+remote-access:
+proto: tcp
+addr: 22
+remote_addr: {{.SSHAddress}}
```
Comment (ghaabor) [inline setupfiles/bootstrap_remote_access.sh.tmpl]: Add an empty line please

File: weekly-cache-update-playbook.yml [hunk 3]
```diff
@@ -64,6 +64,39 @@
...
+    - name: "ngrok-binary: install"
+      unarchive:
+        src: /tmp/ngrok-link-stable-darwin-amd64.tgz
+        dest: /usr/local/bin
+        copy: no
+        mode: 0700
+    - name: "ngrok-binary: remove temp file"
+      file:
+        path: /tmp/ngrok-link-stable-darwin-amd64.tgz
+        state: absent
```
Comment (ghaabor) [inline weekly-cache-update-playbook.yml]: please create `/etc/ngrok` directory like we did in the android-packer repo

File: setupfiles/bootstrap_remote_access.sh.tmpl (hunk 1 above)
Comment (bitvirago) [reply setupfiles/bootstrap_remote_access.sh.tmpl]: fix

File: setupfiles/bootstrap_remote_access.sh.tmpl (hunk 2 above)
Comment (bitvirago) [reply setupfiles/bootstrap_remote_access.sh.tmpl]: fix

File: weekly-cache-update-playbook.yml (hunk 3 above)
Comment (bitvirago) [reply weekly-cache-update-playbook.yml]: fix

File: weekly-cache-update-playbook.yml [hunk 4]
```diff
@@ -64,6 +64,49 @@
...
+        state: directory
+        owner: vagrant
+        group: wheel
+        mode:  "0755"
+
+    - name: "Ensure /etc/ngrok exists"
+        file:
+          path: /etc/ngrok
+          state: directory
+          owner: vagrant
```
Comment (ghaabor) [inline weekly-cache-update-playbook.yml]:
```suggestion
          owner: root
```

**Pull request merged**